internal/services/logic/logic_app_workflow_resource.go

```diff
@@ -161,6 +198,45 @@ func resourceLogicAppWorkflow() *pluginsdk.Resource {
 				},
 			},
 
+			"identity": {
```

Comment:
can we update this to use the common schema from [the `identity` package](https://github.com/hashicorp/terraform-provider-azurerm/tree/main/internal/identity) here?

Reply:
Updated